examples/src/main/java/com/datastax/oss/driver/examples/json/codecs/JacksonJsonCodec.java

```diff
@@ -52,28 +52,28 @@
  */
 public class JacksonJsonCodec<T> implements TypeCodec<T> {
 
-  private final ObjectMapper objectMapper;
+  private final JsonMapper jsonMapper;
   private final GenericType<T> javaType;
 
   /**
    * Creates a new instance for the provided {@code javaClass}, using a default, newly-allocated
-   * {@link ObjectMapper}.
+   * {@link JsonMapper}.
    *
    * @param javaClass the Java class this codec maps to.
    */
   public JacksonJsonCodec(Class<T> javaClass) {
-    this(javaClass, new ObjectMapper());
+    this(javaClass, JsonMapper.builder().build());
   }
 
   /**
    * Creates a new instance for the provided {@code javaClass}, and using the provided {@link
-   * ObjectMapper}.
+   * JsonMapper}.
    *
    * @param javaClass the Java class this codec maps to.
    */
-  public JacksonJsonCodec(Class<T> javaClass, ObjectMapper objectMapper) {
+  public JacksonJsonCodec(Class<T> javaClass, JsonMapper jsonMapper) {
```

Comment:
I know this is a public method signature change, but it is in the `examples` project, so I figured it's not a big deal.